data/themes/classic/style.css

```diff
@@ -24,8 +24,8 @@ AutomationEditor {
 
 	qproperty-graphColor: rgba(153, 175, 255, 200);
 	qproperty-scaleColor: qlineargradient(spread:reflect,
-                     x1:0, y1:0.5, x2:1, y2:0.5,
-                     stop:0 #333, stop:1 #202020);
+				x1:0, y1:0.5, x2:1, y2:0.5,
```

Comment:
These lines don't appear to have been changed and the formatting is inconsistent.